project/src/pages/add-review-screen/add-review-screen.tsx

```diff
@@ -0,0 +1,98 @@
+function AddReviewScreen(): JSX.Element {
+  return (
+    <section className="film-card film-card--full">
+      <div className="film-card__header">
+        <div className="film-card__bg">
+          <img src="img/bg-the-grand-budapest-hotel.jpg" alt="The Grand Budapest Hotel" />
+        </div>
+
+        <h1 className="visually-hidden">WTW</h1>
+
+        <header className="page-header">
+          <div className="logo">
+            <a href="main.html" className="logo__link">
+              <span className="logo__letter logo__letter--1">W</span>
+              <span className="logo__letter logo__letter--2">T</span>
+              <span className="logo__letter logo__letter--3">W</span>
+            </a>
+          </div>
+
+          <nav className="breadcrumbs">
+            <ul className="breadcrumbs__list">
+              <li className="breadcrumbs__item">
+                <a href="film-page.html" className="breadcrumbs__link">The Grand Budapest Hotel</a>
+              </li>
+              <li className="breadcrumbs__item">
+                <a className="breadcrumbs__link">Add review</a>
```

Comment:
```suggestion
                <a href="#add-review" className="breadcrumbs__link">Add review</a>
```
Когда проект развивается и количество файлов растет, такие долги перед линтером накапливаются. Если сообщений литнера становится много, то мы перестаем на них обращать внимание и рискуем пропусить сообщения более серьезные.

Держи код в чистоте.